training_data_collector/main/image_enrichment.py

```diff
@@ -0,0 +1,42 @@
+import cv2
+import random
+
+def enrich_image(element, image):
```

Comment:
gonna get you to walk me through this over a call instead of peppering comments.